sdk/template/azure-sdk-template/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Release History
+## 1.2.1-beta.3 (2020-09-08)
```

Comment:
Should probably be 'unreleased'

Reply:
This is how update-version.py did. I am not sure why.

Reply:
I change it back manually. Will take a look why the script not return me the right thing

Reply:
I suspect there might be another options that is needed to make this unreleased when bumping the version.  Did you do the same command as in https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/templates/stages/archetype-java-release.yml#L235?

Reply:
I did the set-version step right, but when execute update version step. I did not put the parameter --sr
python eng/versioning/update_versions.py --update-type library --build-type ${{ parameters.SDKType }} --avi

Reply:
Tried put the --sr back. It works as expected.